cli/src/test/java/org/openapitools/openapistylevalidator/cli/MainTest.java

```diff
@@ -0,0 +1,68 @@
+package org.openapitools.openapistylevalidator.cli;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Options;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.openapitools.openapistylevalidator.styleerror.StyleError;
+import org.opentest4j.MultipleFailuresError;
+
+import java.util.List;
+
+public class MainTest {
+    private static final DefaultParser parser = new DefaultParser();
+
+    @Test
+    void wihtoutOptionFile() throws Exception {
```

Comment:
What naming convention should we follow for the tests?
The one I had started were using the "should" so they kind of document the app.
For example: `goodUnderscoreCaseShouldReturnTrue`

But I'm open for another naming convention because we don't have a lot of them right now. Let me know what you think